.pre-commit-config.yaml

```
@@ -0,0 +1,69 @@
+      - id: no-duplicate-scripts
+        name: Check for duplicate scripts
+        entry: python scripts/git-hooks/pre-commit-hygiene.py
+        language: system
+        pass_filenames: false
+        files: '^scripts/.*\.py$'
+
+      - id: organized-imports
+        name: Check import organization
+        entry: python -m isort --check-only --diff
+        language: system
+        types: [python]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)